roles/ampliseq/tasks/main.yml

```diff
@@ -40,7 +40,7 @@
 
 - name: Set nf-core/ampliseq Bismark alias
   lineinfile: dest="{{ ngi_pipeline_conf }}/{{ item.script }}"
-              line="alias ampliseq='nextflow run {{ ampliseq_dest }}/ -c {{ ngi_pipeline_conf }}/ampliseq_{{ item.site }}.config -profile uppmax'"
+              line="alias ampliseq='nextflow run {{ ampliseq_dest }}/ -c {{ ngi_pipeline_conf }}/ampliseq_{{ item.site }}.config -c {{ ngi_pipeline_conf }}/nextflow_irma_{{ item.site }}.config'"
```

Comment:
The order of configs is sometimes important, so I think it's better to have the generic one first..

```yaml
line="alias ampliseq='nextflow run {{ ampliseq_dest }}/ -c {{ ngi_pipeline_conf }}/nextflow_irma_{{ item.site }}.config -c {{ ngi_pipeline_conf }}/ampliseq_{{ item.site }}.config'"
```